includes/polling/applications/fail2ban.inc.php

```diff
@@ -16,23 +16,20 @@
 $bannedStuff = explode("\n", $f2b);
 
 $total_banned=$bannedStuff[0];
-$firewalled=$bannedStuff[1];
 
-$rrd_name = array('app', $name, $app_id);
+$rrd_name = array('app', $name, $app_id, 'totalbanned');
 $rrd_def = RrdDefinition::make()
-    ->addDataset('banned', 'GAUGE', 0)
-    ->addDataset('firewalled', 'GAUGE', 0);
+    ->addDataset('banned', 'GAUGE', 0);
```

Comment:
Does this break existing graphs as it would expect 2 entries for people who ran the code before this change?

Reply:
Talked to Murrant about this and ended up changing the name of the RRD as otherwise people would need to remove the old ones.

Reply:
Why not just leave the old file in place and update the firewalled value with 0 and remove from the webui so you never see it. Changing the filename results in people losing the historic data. @murrant - what do you think?

Reply:
Was worried about bloat with RRDs with pointless data in them. But then again it is very minor.

Decided to rework this a bit.

Just shoving U into the old unused value. I've tested it and it works. :)